chain/epoch-manager/src/adapter.rs

```diff
@@ -88,6 +88,9 @@ pub trait EpochManagerAdapter: Send + Sync {
         parent_hash: &CryptoHash,
     ) -> Result<EpochHeight, EpochError>;
 
+    /// Get next epoch id given hash of the current block.
+    fn get_next_epoch_id(&self, block_hash: &CryptoHash) -> Result<EpochId, EpochError>;
```

Comment:
There are some of these functions that exist in EpochManager but not in EpochManagerAdapter. So these needed to be added here.